Log/lambda_function.py

```diff
@@ -62,28 +58,44 @@ def lambda_handler(event, context):
     #Add custom tags here by adding new value with the following format "key1:value1, key2:value2"  - might be subject to modifications
     metadata[DD_CUSTOM_TAGS] = "functionname:" + context.function_name+ ",memorysize:"+ context.memory_limit_in_mb
 
+    try:
+        logs = generate_logs(event)
+        for log in logs:
+            s = safe_submit_log(s, log)
+    except Exception as e:
+        print('Uncaught exception: {}'.format(str(e)))
```

Comment:
This exception is caught, you should change `Uncaught` to `Unexpected`. I would also print `event` for debugging purpose.